app/views/users/show.html.erb

```diff
@@ -1 +1,12 @@
-<h1>user show placeholder</h1>
\ No newline at end of file
+
+<div class="container">
+<%= render partial: 'user', locals: { user: @user } %>
+<h4 class="text-center">Bio</h4>
+<p class="text-center"><%=@user.bio%></p>
+<% @user.recent_posts.each do |post|%>
+ <%= render partial: 'posts/post', locals: { post: post } %>
```

Comment:
- [ ] The `recent_posts` method should return the three most recent posts of a certain user. Currently, you are returning the first three posts of the user. I would advise you to change the logic for the recent_posts in your `app/models/user.rb` file. 👍 😎 

#### Hint: 
```ruby
def recent_posts
  posts.order(created_at: :desc).limit(3)
end
```